Build.cs

```diff
@@ -412,63 +444,64 @@ private bool ValidateROM(string FileName)
 
         private void MakeROM(string InFile, string FileName, BackgroundWorker worker)
         {
+            if(Settings.GenerateROM)
```

Comment:
This line is unnecessary. It should be (and already is) checked where this function is called.